### ISSUE TITLE
[BUG] Broken Types

### ISSUE THREAD

**Issue by hyperupcall** [markmap/markmap#186]
- [x] I have searched for existing issues that already reported this problem and found none
- [ ] The bug is present in [REPL](https://markmap.js.org/repl)
  

**Describe the bug**

Currently, types cannot be resolved for at least `markmap-lib`, and very likely other markmap packages.

**To Reproduce**

Use the library as document (import anything from markmap). Example:

```typescript
import { Transformer } from 'markmap-lib'
import { Markmap } from 'markmap-view/dist/index.esm'
```

**Expected behavior**

`Transformer` and `Markmap` are properly typed and no red underlines

**Actual behavior**

There are red underlines when importing from markmap packages. Screenshot: 
![image](https://github.com/markmap/markmap/assets/24364012/a2ef9de5-0eb5-4132-91df-f0790183370d)

**Environment:**

- OS: Linux
- Browser: N/A
- Browser Version: N/A

**Comment by gera2ld** [comment]
It works for me in VSCode and [Stackblitz](https://stackblitz.com/edit/markmap?file=index.tsx).
Could you provide a minimal reproduction?

**Comment by hyperupcall** [comment]
Thank you for your patience, I have uploaded a new repro at https://github.com/hyperupcall/repro-markmap-missing-types

The only error is from the `markmap-lib`. `markmap-view` is okay

**Comment by gera2ld** [comment]
Fixed in c3207d2885b9a4d4de1f8a68d57b4eb4b47c0e0b

**Comment by hyperupcall** [comment]
Thank you for your work! I really appreciate  markmap
